### PR TITLE
chore(ci): add automated tests on GKE & fix AKS

### PR DESCRIPTION
**Pull request by zdrapela**
## Description

- Establish connection to GKE cluster using `gcloud` CLI.
- Add setup scripts for GKE deployments.
- Move AKS and GKE deployment functions out of the `openshift-ci-test.sh`.
- Replace AKS-specific namespace with K8S namespace/playwright project that both AKS and GKE can use.
- Change the creation of ConfigMaps.
- Fix AKS tests

The GKE nightly job will be established in this PR: https://github.com/openshift/release/pull/58943

## Which issue(s) does this PR fix

- Fixes [RHIDP-4552](https://issues.redhat.com/browse/RHIDP-4552), [RHIDP-4551](https://issues.redhat.com/browse/RHIDP-4551) and [RHIDP-4861](https://issues.redhat.com/browse/RHIDP-4861)

## PR acceptance criteria

✅ **AKS and GKE tests passed while running with a local `showcase-e2e-runner` built from `.ibm/images/Dockerfile`.**

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by subhashkhileri — CHANGES_REQUESTED** [review]

File: .ibm/pipelines/openshift-ci-tests.sh [hunk 1]
```diff
@@ -450,10 +430,23 @@ main() {
   echo "Log file: ${LOGFILE}"
   set_cluster_info
   source "${DIR}/env_variables.sh"
+  if [[ "$JOB_NAME" == *periodic* ]]; then
```
[the remark this answers is not in the document]
Comment (zdrapela) [reply .ibm/pipelines/openshift-ci-tests.sh]: Good catch. I added a fix

**Pull request merged**